### PR TITLE
C demo

### PR DESCRIPTION
**Pull request by ksyeo1010**
- changed app_id to access_key
- changed mic demo to use analog instead of threshold

### CODE REVIEW

File: demo/c/cobra_demo_mic.c [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <math.h>
```
Comment (kenarsa) [inline demo/c/cobra_demo_mic.c:16]: move up

File: demo/c/cobra_demo_mic.c [hunk 2]
```diff
@@ -108,24 +111,36 @@ void show_audio_devices(void) {
...
 int main(int argc, char *argv[]) {
-    if (argc != 5) {
+    if (argc != 4) {
         if ((argc == 2) && (strcmp(argv[1], "--show_audio_devices") == 0)) {
             show_audio_devices();
             return 0;
         } else {
             fprintf(stderr, "usage : %s --show_audio_devices\n"
-                            "        %s library_path app_id threshold audio_device_index\n", argv[0], argv[0]);
+                            "        %s library_path access_key threshold audio_device_index\n", argv[0], argv[0]);
```
Comment (kenarsa) [inline demo/c/cobra_demo_mic.c:134]: remove `threshold`

File: demo/c/cobra_demo_mic.c [hunk 3]
```diff
@@ -84,6 +85,8 @@ static void print_dl_error(const char *message) {
 }
 
 static volatile bool is_interrupted = false;
+static const float alpha  = 0.25f;
```
Comment (mrrostam) [inline demo/c/cobra_demo_mic.c:88]: decrease `alpha` as right now it seems that there is a noticeable delay for recognizing voice activities. I've tried 0.15 and it was good to my eye but feel free to experiment more

File: demo/c/cobra_demo_mic.c [hunk 4]
```diff
@@ -224,20 +238,9 @@ int main(int argc, char *argv[]) {
...
-            if (!is_clean) {
-                fprintf(stdout, "\r                  \r");
-            }
-            is_clean = true;
-        }
-
-        fflush(stdout);
+        print_analog(is_voiced);
     }
+    fprintf(stdout,"\n");
```
Comment (mrrostam) [inline demo/c/cobra_demo_mic.c:243]: missing space after comma

**Pull request merged**